releaseNotes/JavaScriptProvider.md

```diff
@@ -2,6 +2,18 @@
 
 [Source code][source_code] | [Package (npm)][package] | [Samples][samples]
 
+## 1.1.0 - August 13, 2024
+
+### Enhancements
+
+- Added support for loading feature flags from Azure App Configuration. [#65](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/65)
+
+## 1.0.1 - July 24, 2024
+
+### Bug Fixes
+
+- Fixed a possible reference error when sending request in Web Worker environment. [#81](https://github.com/Azure/AppConfiguration-JavaScriptProvider/issues/81)

```

Comment:
Is this accurate?

Fixed a bug that caused the error ‘ReferenceError: WorkerNavigator is not defined’ when running the application in a browser.

Reply:
Not accurate. To clarify:

It was a bug on "detecting whether the environment is WebWorker". On non-WebWorker environment, it mistakenly threw an error. On WebWorker environment, it was mistakenly regarded as normal browser env.

Reply:
> Is this accurate?

No. I was wrong.